packages/airnode-protocol/src/index.ts

```diff
@@ -66,11 +67,18 @@ export {
   RequesterAuthorizerWithAirnodeAddresses,
   AirnodeRrpFactory,
   AccessControlRegistryFactory,
+  RrpBeaconServerFactory,
```

Comment:
You don't need to import a thing just to export it, you can export it directly.

```suggestion
export { RrpBeaconServer__factory as RrpBeaconServerFactory } from './contracts';
```

Reply:
Right. I just tried to apply the same convention already being use (lines 13-19)